src/services/users/userService.ts

```diff
@@ -0,0 +1,76 @@
+import { v4 as uuid } from "uuid";
+import { Op, literal } from "sequelize";
+import sequelize from "../../data-acess/db";
+import { User as UserModel } from "../../models/User";
+import { User, UserInternalProps } from "../../types/user";
+
+export default class UserService {
+    async getUserById(id: string) {
+        await sequelize.sync();
```

Comment:
Here is no sense to sync the db on any call. It makes your performance to be worse. The sync should be done once after all models initialization or sync models only if it's supposed to be not initialized all at once

Reply:
Fixed into https://github.com/SorokinTim/nodejs-gmp/pull/6/commits/a3a56e47c667ff530cd567600bbbf06ec3e0f01d commit